### PR TITLE
Add missing changelog entries from the past week.

### PR DESCRIPTION
**Pull request by karst**
🐍

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -12,6 +14,8 @@
 - Improved: [#18970] Trying to load a non-park save will now display a context error.
 - Improved: [#18975] Add lift sprites for 60 steep hills on the wooden roller coaster.
 - Improved: [#19044] Added special thanks to RMC and Wiegand to the About page.
+- Improved: [#19131] Track missing objects when selecting scenery groups in console.
```
Comment (trzejos) [inline distribution/changelog.txt:17]: I'm not sure end users are affected by this one too much. It just uses LOG_ERROR when selecting a scenery group, and I don't think that gets logged anywhere unless you start up a debug build from the command line. Maybe this one could be reworded to say "Log missing ..."

File: distribution/changelog.txt [hunk 2]
```diff
@@ -25,6 +29,7 @@
 - Fix: [#19112] Clearing the last character in the Object Selection filter does not properly reset it.
 - Fix: [#19114] [Plugin] GameActionResult does not comply to API specification.
 - Fix: [#19136] SV6 saves with experimental RCT1 paths not imported correctly.
+- Removed: [project#1] Snakes.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:32]: Refactors are never put in the log. Also we didn't remove Herobrine.

**Comment by Gymnasiast** [comment]
Don't touch the PR yet, I'll make some changes myself.

File: distribution/changelog.txt (hunk 2 above)
Comment (karst) [reply distribution/changelog.txt:32]: Dang it, those darn herobrines.

File: distribution/changelog.txt (hunk 1 above)
Comment (duncanspumpkin) [reply distribution/changelog.txt:17]: You can see these logs if you run the console build (.com) that is distributed with all versions. I think therefore it does make sense to have this entry.

File: distribution/changelog.txt (hunk 2 above)
Comment (tupaschoal) [inline distribution/changelog.txt:32]:
```suggestion
```

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**